app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def most_recent_comments
+    comments.order('created_at DESC').limit(5)
+  end
```

Comment:
- [OPTIONAL] It would be nice to make the `update_post_counter` method private so it serves as a helper only inside this class